frontend/src/screens/admin/grid.tsx

```diff
@@ -0,0 +1,31 @@
+import { React, Box, styled } from '@common'
+import { BoxProps } from 'boxible'
+
+export const Grid = styled.div({
+    display: 'grid',
+    height: '100vh',
+    width: '100vw',
+    gridTemplateColumns: 'minmax(auto, 200px) 1fr',
+    gridTemplateRows: 'auto 1fr auto',
+    overflow: 'hidden',
+    gridTemplateAreas: `
```

Comment:
This is super cool, i didn't know this was a thing. read up on it a little more and wonder if down the line we could build out the whole site's structure like this!